[2024-03-02T19:15:04.927+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-03-02T19:15:04.944+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-02-13T00:00:00+00:00 [queued]>
[2024-03-02T19:15:04.944+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2024-03-02T19:15:05.072+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_html_data> on 2024-02-13 00:00:00+00:00
[2024-03-02T19:15:05.078+0000] {standard_task_runner.py:60} INFO - Started process 1300 to run task
[2024-03-02T19:15:05.081+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_ncei_data', 'download_html_data', 'scheduled__2024-02-13T00:00:00+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/hi.py', '--cfg-path', '/tmp/tmpp0rsjkn8']
[2024-03-02T19:15:05.083+0000] {standard_task_runner.py:88} INFO - Job 114: Subtask download_html_data
[2024-03-02T19:15:05.270+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_ncei_data.download_html_data scheduled__2024-02-13T00:00:00+00:00 [running]> on host a890577b40a7
[2024-03-02T19:15:05.493+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='fetch_ncei_data' AIRFLOW_CTX_TASK_ID='download_html_data' AIRFLOW_CTX_EXECUTION_DATE='2024-02-13T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-13T00:00:00+00:00'
[2024-03-02T19:15:05.496+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-02T19:15:05.498+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "curl {{ dag_run.conf.get('base_url', params.base_url) }}2001/ --create-dirs -o /tmp/html/2001.html"]
[2024-03-02T19:15:05.509+0000] {subprocess.py:86} INFO - Output:
[2024-03-02T19:15:05.511+0000] {subprocess.py:93} INFO - /usr/bin/bash: -c: line 1: syntax error near unexpected token `('
[2024-03-02T19:15:05.512+0000] {subprocess.py:93} INFO - /usr/bin/bash: -c: line 1: `curl {{ dag_run.conf.get('base_url', params.base_url) }}2001/ --create-dirs -o /tmp/html/2001.html'
[2024-03-02T19:15:05.512+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2024-03-02T19:15:05.529+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2024-03-02T19:15:05.534+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=fetch_ncei_data, task_id=download_html_data, execution_date=20240213T000000, start_date=20240302T191504, end_date=20240302T191505
[2024-03-02T19:15:05.679+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 114 for task download_html_data (Bash command failed. The command returned a non-zero exit code 2.; 1300)
[2024-03-02T19:15:05.697+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-02T19:15:05.731+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
